.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["@babel/preset-env", "@babel/preset-react"]
```

Comment:
I know Babel is a JS compiler but not too much else about it. Does it make sense to have the react preset at the top level even though it only applies to the front end? Also, if we are using Typescript for the back end we should think about adding that preset because I think it makes compiling a fair bit easier.

Reply:
I think this is fine. The believe presets definition here is needed because it doesn't check package.json for the necessary dependencies. It shouldn't run 'ES6 to 5' and React conversions where it isn't supposed to.